project/Common.scala

```diff
@@ -35,7 +35,9 @@ object CommonSettings extends AutoPlugin {
       scalaVersion := Dependencies.ScalaVersion,
       run / javaOptions ++= {
         sys.props.collect { case (key, value) if key.startsWith("akka") => s"-D$key=$value" }(breakOut)
-      }
+      },
+      // to get latest versions
+      resolvers += "akka-http-snapshot-repository" at "https://oss.sonatype.org/content/repositories/snapshots",
```

Comment:
I don't think we can use sonatype snapshots in the sdk, because existing builds (pom.xml) doesn't have that snapshot repo configured. We probably need a real milestone on maven central.

Or we could wait with merging/releasing the SDK.

```
Failed to execute goal on project valueentity-counter: Could not resolve dependencies for project com.example:valueentity-counter:jar:0.1: Could not find artifact com.lightbend.akka.grpc:akka-grpc-runtime_2.13:jar:2.0.0-30-fa341ccd-SNAPSHOT
```

Reply:
ah, that was what you said